src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = new User(email, password);
+        if (new UserService().findByEmail(email) != null
+                && user.getPassword().equals(new UserService().findByEmail(email).getPassword())) {
```

Comment:
moreover, you create it twice per each method call. You should create one per each method _instance_